core/templates/pages/blog-dashboard-page/blog-post-editor/blog-post-editor.component.ts

```diff
@@ -116,6 +116,7 @@ export class BlogPostEditorComponent implements OnInit {
     }
   }
 
+
```

Comment:
Here and elsewhere, please remove the extra newlines.

Reply:
> Here and elsewhere, please remove the extra newlines.

@kevintab95 i removed the extra newline. which was inserted by me.